cassiopeia/solver/NeighborJoiningSolver.py

```diff
@@ -1,5 +1,5 @@
 """
-This file stores a subclass of DistanceSolver, NeighborJoining. The
+This file stores a subclass of CCPhyloSolver, NeighborJoiningSolver. The
```

Comment:
This is no longer accurate, right? Here `NeighborJoiningSolver` inherits from `DistanceSolver`.